### DISCUSSION TITLE
Add an option to disable close up zoom in

### DISCUSSION THREAD

**Discussion by Terraformer9x** [OpenRCT2/OpenRCT2#22867] [Feature requests]
I noticed with the latest builds that the OpenGL close up zoom in is on every render mode now, I was wondering if there can be an option to disable it? I find it to be really disorienting and I don't like that I can scroll past the standard zoom in.

**Comment by J-PIE-314** [comment]
#23152 is a more option-filled version of this request. #15793 is similar.
